src/commonMain/kotlin/uk/gov/hmrc/helptosavecalculator/FirstBonusTermCalculation.kt

```diff
@@ -87,7 +87,7 @@ internal class FirstBonusTermCalculation {
     }
 
     fun calculateMonthsLeftInScheme(input: FirstBonusInput): Pair<Int, Int> {
-        val startDate = input.accountStartDate.convertToDateTime()
+        val startDate = input.thisMonthEndDate.convertToDateTime()
```

Comment:
Lets change `startDate` as it's no longer that ;)

Reply:
thanks.